<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_seacher">
          <property name="text">
           <string>搜索串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="com_choose"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_open">
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>波特率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="box_baud">
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>460800</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stopBitLabel">
          <property name="text">
           <string>停止位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stopBitBox">
          <item>
           <property name="text">
            <string>One</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OneAndHalf</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Two</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="parityBitlabel">
          <property name="text">
           <string>校验位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="parityBitBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mark</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dataBitlabel">
          <property name="text">
           <string>数据位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dataBitBox">
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>238</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="imagbar">
    <property name="title">
     <string>上位机</string>
    </property>
   </widget>
   <addaction name="imagbar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionimage"/>
   <addaction name="actionwave"/>
   <addaction name="actioninstruct"/>
   <addaction name="actionparameter"/>
   <addaction name="separator"/>
   <addaction name="setting"/>
   <addaction name="actionhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionimage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/m5.ico</normaloff>:/image/m5.ico</iconset>
   </property>
   <property name="text">
    <string>图传</string>
   </property>
   <property name="toolTip">
    <string>image</string>
   </property>
  </action>
  <action name="actionwave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/m7.ico</normaloff>:/image/m7.ico</iconset>
   </property>
   <property name="text">
    <string>示波器</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actioninstruct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/m10.ico</normaloff>:/image/m10.ico</iconset>
   </property>
   <property name="text">
    <string>指令</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/m9.ico</normaloff>:/image/m9.ico</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionparameter">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/m19.ico</normaloff>:/image/m19.ico</iconset>
   </property>
   <property name="text">
    <string>参数</string>
   </property>
   <property name="toolTip">
    <string>参数</string>
   </property>
  </action>
  <action name="setting">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/m20.ico</normaloff>:/image/m20.ico</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
